libraries/testkit/sgaccel.py

```diff
@@ -59,14 +60,19 @@ def start(self, config):
             "revs_limit": "",
             "num_index_replicas": "",
             "sg_use_views": "",
-            "logging": "",
```

Comment:
Won't you get a KeyError if you remove this key?

Reply:
actually , my if and else, both conditions are creating key there, so it does not required to assign blank string here